cea/technologies/constants.py

```diff
@@ -79,3 +79,8 @@
 
 #natural gas conncetion
 GAS_CONNECTION_COST = 15.5 / 1000  # CHF / W, from  Energie360 15.5 CHF / kW
+
+# Thermal Network
+TYPE_MAT_DEFAULT = 'T1'
```

Comment:
Unclear to me what this variable means, is it material type? Perhaps add a comment to make more clear. Pipe diameter is self explanatory, but I would add the units is being considered, just for documentations

Reply:
Yes. this is meant to declare the type of pipes (material and construction) that is used for the network. This is based on the existing thermal network code. I added a comment in the constants-file.
However,  I just realised, that the pipe type is currently a bit meaningless. The only method this could play into is the `calc_linear_thermal_loss_coefficient`-function from `simplified_thermal_network.py`, but the function assumes a constant construction (steel pipe + calcium silicate insulation). So, there is currently no database in CEA specifying the properties of these types, but that could be added in the future.

Reply:
Got it! Could it be relevant for the LCA parts, perhaps?